web-app/templates/book/createBookView.html

```diff
@@ -0,0 +1,80 @@
+<form name="createBookForm" ng-submit="createNewBook()">
+
+    <div class="form-group">
+        <label>Name:</label>
+        <div style="display: flex">
+            <input required ng-minlength="3" ng-model="name"
```

Comment:
move "3" to controller and set like constant:
For Example:
// ../create.book.controller.js
...
{
this.NAME-FIELD-MIN-LENGHT = 3'
}